src/vim.rs

```diff
@@ -276,15 +276,19 @@ impl State {
         Ok(())
     }
 
-    pub fn setqflist(&mut self, list: &[QuickfixEntry]) -> Result<()> {
-        if self.call::<_, u8>(None, "setqflist", json!([list, "r"]))? != 0 {
+    pub fn setqflist(&mut self, list: &[QuickfixEntry], replace: bool, title: &str) -> Result<()> {
```

Comment:
I'd suggest making the parameter a bit more extensible, i.e., replace `replace: bool` to `action: &str`.

Reply:
done